src/unit_tests/syscheckd/test_create_db.c

```diff
@@ -986,7 +986,7 @@ static void test_fim_check_ignore_strncasecmp(void **state) {
 
     expect_string(__wrap__mdebug2, formatted_msg, debug_msg);
 
-    ret = fim_check_ignore("/EtC/dumPDateS");
+    ret = fim_check_ignore("/EtC/dumPDateS",FIM_REGULAR);
```

Comment:
Consider adding space after the comma for readability.  Please, also check the following instances.